src/server/page.html.jsx

```diff
@@ -121,7 +129,7 @@ export async function renderPage(request, reply) {
           media="(min-device-width: 1024px) and (max-device-width: 1024px) and (-webkit-min-device-pixel-ratio: 2) and (orientation: portrait)"
         />
 
-        <style>{globalStyles.getStyles()}</style>
+        <style dangerouslySetInnerHTML={{ __html: globalStyles.getStyles() }} />
```

Comment:
why this change?

Reply:
because of the general rule I added - `[tabindex="-1"]`. I didnt wan't the quotation marks to be escaped.

Reply:
feels like a hammer solution though. I'm confident there must be a way to write the rule that doesn't require this.